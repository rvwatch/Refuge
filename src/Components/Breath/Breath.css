
.box-breathing {
  width: 50vmin;
  height: 50vmin;
  margin-top: -25vmin;
  margin-left: -25vmin;
  position: absolute;
  top: 50%;
  left: 50%;
  border-radius: 50%;
  border: 5px solid #87cefa;
}

.box-breathing::after {
  box-sizing: border-box;
  font-size: 30px;
  padding-top: 18vmin;
  color: grey;
  width: 44vmin;
  height: 44vmin;
  margin-top: -22vmin;
  margin-left: -22vmin;
  position: absolute;
  top: 50%;
  left: 50%;
  border-radius: 50%;
  content: 'breathe';
  text-align: center;
  -webkit-animation: box-breath 5.5s infinite;
          animation: box-breath 5.5s infinite;
}

@-webkit-keyframes box-breath {
  0%, 75%, 100% {
    -webkit-transform: scale(0.3);
            transform: scale(0.3);
            background-color: #72ffcb;
  }
  25%, 50% {
    -webkit-transform: scale(1);
            transform: scale(1);
            background-color: #87cefa;
  }
}
