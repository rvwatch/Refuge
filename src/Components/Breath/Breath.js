import React from 'react';
import {Link} from 'react-router-dom';
import './Breath.css';

export const Breath = () => {
  return (
    <section className='breathe-wrap'>
      
      <div className='box-breathing'></div>
      <header className='excercise-nav'>
        <Link to='/' className='prev'>dashboard</Link>
        <Link to='/mindfulness' className='next'>meditate</Link>
      </header>
    </section>
  ); 
};

