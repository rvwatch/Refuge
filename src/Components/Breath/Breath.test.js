import  React from 'react';
import {Breath} from './Breath';
import {shallow} from 'enzyme';

describe('Breath', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<Breath />);
    expect(wrapper).toMatchSnapshot();
  });
});
