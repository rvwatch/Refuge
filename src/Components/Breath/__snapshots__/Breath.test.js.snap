// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breath should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="box-breathing"
/>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "className": "box-breathing",
      },
      "ref": null,
      "rendered": null,
      "type": "div",
    },
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="box-breathing"
/>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "box-breathing",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
